Main.java

```diff
@@ -9,7 +11,40 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // Creat a Scanner for user input
+    Scanner input = new Scanner(System.in);
+    
+    // asks the users to input the first coordinates of x
+    System.out.println("Please enter in the x coordinate of the first point");
+
+    // gets the users input
+    int x1 = input.nextInt();
+
+    // asks the users to input the first coordinates of y
+    System.out.println("Please enter in the y coordinate of the first point");
+
+    // gets the users input
+    int y1 = input.nextInt();
+
+    // asks the users to input the second coordinates of x
+    System.out.println("Please enter in the x coordinate of the second point");
+
+    // gets the users input
+    int x2 = input.nextInt();
+    
+    // asks the users to input the second coordinates of y
+    System.out.println("Please enter in the y coordinate of the second point");
+
+    // gets the users input
+    int y2 = input.nextInt();
+
+    // calculates the users slope
+    int answer = (y2 - y1) / (x2 - x1);
```

Comment:
Can you have a point where the x or y value has a decimal place in it? These should probably be doubles instead of ints. We can tell this because in the sample output, he slope is shown as 2.0 instead of just 2.